<!DOCTYPE html>
<html lang="en">
<head>
  <title>  Previous work
 - Hadrien Mary - PhD student in Biology and Bioinformatic</title>

  <meta charset="utf-8" />
  <link rel="icon" type="image/png" href="/theme/img/favicon.png" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0">

  <meta name="google-site-verification" content="XAa3tuYNxza7nP5i5DTfsTbNftieMBsWxG9Q-kzBngw" />
  <meta name="description" content="Hadrien Mary, Postdoc in Biology.">

  <link href="/theme/css/bootstrap.min.css" rel="stylesheet" media="screen">

  <link href='http://fonts.googleapis.com/css?family=Crimson+Text' rel='stylesheet' type='text/css'>
  <link href='http://fonts.googleapis.com/css?family=Yanone+Kaffeesatz' rel='stylesheet' type='text/css'>
  <link rel="stylesheet" href="/theme/css/main.css" type="text/css" />
</head>

<body id="index" class="home">
<div class="container-fluid">
  <header id="banner" class="body">
    <h1>
      <img src="/theme/img/header.png"/>
      <a href="/">Hadrien Mary       <strong class="hidden-xs">Postdoc in Biology</strong></a>
    </h1>

    <nav class="navbar navbar-default" role="navigation">
      <div class="container-fluid">

        <div class="navbar-header">
            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#navigationbar">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
             </button>
        </div>

        <div class="collapse navbar-collapse" id="navigationbar">
          <ul class="nav navbar-nav">

              <li>
                <a href="/">Research</a>
              </li>
              <li class="active">
                <a href="/previous-work">Previous Work</a>
              </li>
              <li>
                <a href="/visualization">Visualization</a>
              </li>
              <li>
                <a href="/softwares">Softwares</a>
              </li>
              <li>
                <a href="/cv">CV</a>
              </li>
              <li>
                <a href="/contact">Contact</a>
              </li>
              <li>
                <a href="/credits">Credits</a>
              </li>
          </ul>
        </div><!-- /.navbar-collapse -->
      </div><!-- /.container-fluid -->
    </nav>

  </header><!-- /#banner -->

  <div id="content-container" class="body">
  <section id="content">
    <h1 class="entry-title">Previous work</h1>
    
    <h2>2012-2015 - PhD project : Analysis and modeling of chromosome dynamics during mitosis in fission yeast</h2>
<blockquote class="blockquote-reverse">
  <p>« The dream of every cell is to become two cells. »</p>
  <footer>François Jacob, 1974</footer>
</blockquote>

<p>(my thesis can be downloaded as pdf <a href="/pdfs/thesis.pdf">here</a>, only in french)</p>
<figure class="float right">
  <img src="/images/mitosis.png" alt="Different phase of mitosis" width="40%" >
  <figcaption>Fig 1. Different phase of mitosis (Cheeseman et al., 2008).</figcaption>
</figure>

<p>Mitosis is a highly preserved process in all eukaryotic cells during which the genetic material (chromosomes) is divided in two parts which spread in both daughter cells. This equipartition is crucial for maintaining genetic stability. During this process, chromosomes form a metaphasic plate at the center of the mitotic spindle. Each chromatid is attached to its respective spindle pole (called bipolar attachment) toward which it will move during anaphase.</p>
<p>Chromatids are the indivisible units of genetic material during mitosis just like atoms in a molecule. Originally each of these « objects » are detached and organized in chromosomes territories. All the complexity of mitosis resides in the capture of each chromatid by the spindle pole to exert forces to position them on the metaphase plate before their separation and migration towards their respective poles in anaphase.</p>
<figure>
  <img src="/images/spindle_overview.png" alt="An overview of the mitotic spindle in fission yeast." width="50%" >
  <figcaption>Fig 2. An overview of the mitotic spindle in fission yeast.</figcaption>
</figure>

<p>This step of cell division not only requires complex interaction networks and metabolic signaling pathways just like many other biological processes but also a fine spatio-temporal control of movement and positioning of these big objects relative to cell size.</p>
<p>It is usually accepted that the origin of chromosome movement arises from microtubule dynamics. However, what is less clear is the relative importance of each of these processes regulating chromosome movement: the intrinsic dynamic instability of microtubules or the effect of their associated proteins such as MAPs and kinesins. It is also important to note that the mechanism controlling the transfer of energy between microtubule dynamics and chromosome movement is still largely hypothetical.</p>
<figure>
  <img src="/images/actors.png" alt="Some actors involved in mitosis in fission yeast." width="60%" >
  <figcaption>Fig 3. Some actors involved in mitosis in fission yeast.</figcaption>
</figure>

<p>Moreover, chromosome dynamics during mitosis is regulated by a large number of actors apart from microtubules. Some of them being responsible for MT-kinetochore attachment such as NDC80 and DAM1 complex. While others are involved in the regulation of MT dynamics such as Kinesin-8 and Kinesin-13.</p>
<p>During my PhD, I studied fission yest chromosome dynamic during mitosis. This cellular model has the advantage of sharing many fundamental mechanisms of symmetrically dividing higher eukaryotic cells. I characterized two of these conserved mechanisms: chromosome alignment during metaphase and back and forth movement along the spindle, called chromosome oscillation. By analyzing chromosome trajectories, I showed that both processes are performed through independent mechanisms (Mary, 2015). Moreover, chromosome alignment process, which is still poorly understood, is regulated by Kinesin-8 via a length dependent activity on microtubules. This suggests that Kinesin-8 is able to provide spatial information along the mitotic spindle to properly position chromosomes. Finally, I used a mathematical model (Gay, 2012) of chromosome segregation in order to test quantitatively different hypotheses of chromosome centering process.</p>
<p>This work is thus deciphering the control of movement, attachment and positioning of chromosomes during mitosis and seeks to better understand the biophysical processes controlling mitosis.</p>
<figure>
  <img src="/images/wt.png" alt="In vivo and in silico analysis of chromosome trajectories." width="80%" >
  <figcaption>Fig 4. <em>In vivo</em> and <em>in silico</em> analysis of chromosome trajectories.</figcaption>
</figure>

<h3>References</h3>
<p>Cheeseman, I. M. and Desai, A. (2008). Molecular architecture of the kinetochore-microtubule interface. Nat. Rev. Mol. Cell Biol. 9, 33–46.</p>
<p>Gay, G., T. Courtheoux, C. Reyes, S. Tournier, and Y. Gachet. 2012. A stochastic model of kinetochore-microtubule attachment accurately describes fission yeast chromosome segregation. J. Cell Biol.</p>
<p>Mary, H., Fouchard, J., Gay, G., Reyes, C., Gauthier, T., Gruget, C., Pecreaux, J., Tournier, S. and Gachet, Y. (2015). Fission yeast kinesin-8 controls chromosome congression independently of oscillations. J. Cell Sci. 128, 3720–3730.</p>
<hr>

<h2>2011 - Internship at the Institute for Molecular Bioscience</h2>
<p>I made my internship in 2011 at the <a href="http://www.imb.uq.edu.au" target="_blank">Institute for Molecular Bioscience</a> at the University of Queensland in Australia in the <a href="http://www.imb.uq.edu.au/nick-hamilton" target="_blank">N. Hamilton group</a> (IMB). I was involved in two different projects.</p>
<p>(My internship report is <a href="/pdfs/internship_report.pdf">here</a>.)</p>
<h3>High-throughput image analysis</h3>
<p>I analyzed A431 cells infected by <em>Salmonella</em> and treated with various shRNA. The first step was to detect nucleus (in blue), cytoplasm (in green) and <em>Salmonella</em> (in red) in acquired images.</p>
<figure>
  <img src="/images/imb/17.png" alt="Automatic objects detection for nucleus, cytoplasm and Salmonella" width="50%" >
  <figcaption>Fig 1. Automatic objects detection for nucleus, cytoplasm and Salmonella.</figcaption>
</figure>

<p>The second step was to extract features for each images and conditions. Then I created a visualization (inspired from Collinet et al., 2010) to quickly detect main differences between different conditions.</p>
<figure>
  <img src="/images/imb/profile2.png" alt="Automatic features profile" width="80%" >
  <figcaption>Fig 2. Automatic features profile. The x-axis shows features extracted from images. The y-axis indicates the normalized value for the corresponding feature. On the right panel, conditions can be selected and/or highlighted.</figcaption>
</figure>

<h3>Branching morphogenesis in mouse kidney development</h3>
<p>When I started my work the project was at the beginning and I mainly created various tools to analyze young mouse kidney. Collaborators from Melbourne developed a powerful technics to images mouse kidney in 3D.</p>
<figure>
  <img src="/images/imb/step1.png" alt="3D mouse kidney reconstruction" width="80%" >
  <figcaption>Fig 3. 3D mouse kidney reconstruction.</figcaption>
</figure>

<figure>
  <img src="/images/imb/real.png" alt="Mouse kidney reconstructed as a graph." width="60%" >
  <figcaption>Fig 4. Mouse kidney reconstructed as a graph.</figcaption>
</figure>

<p>Since the project starts in 2011, the team has published various nice papers (Short et al., 2014).</p>
<h3>References</h3>
<p>Collinet, C., Stöter, M., Bradshaw, C. R., Samusik, N., Rink, J. C., Kenski, D., Habermann, B., Buchholz, F., Henschel, R., Mueller, M. S., et al. (2010). Systems survey of endocytosis by multiparametric image analysis. Nature 464, 243–249.</p>
<p>Short, K. M., Combes, A. N., Lefevre, J., Ju, A. L., Georgas, K. M., Lamberton, T., Cairncross, O., Rumballe, B. A., McMahon, A. P., Hamilton, N. A., et al. (2014). Global quantification of tissue dynamics in the developing mouse kidney. Dev. Cell 29, 188–202.</p>
  </section>
  </div>

  <section id="extras" class="body row">
    <div class="blogroll col-md-6">
      <h2>links</h2>
      <ul>
          <li><a href="http://brouhardlab.mcgill.ca/" target="_blank">Brouhard's Lab: my lab</a></li>
          <li><a href="http://cbi-toulouse.fr/eng/equipe-tournier-gachet" target="_blank">Tournier-Gachet Team: my former lab</a></li>
          <li><a href="http://damcb.com/" target="_blank">Guillaume Gay, an independant researcher and a friend</a></li>
      </ul>
</div><!-- /.blogroll -->
    <div class="social col-md-6">
      <h2>be social</h2>
      <ul>
          <li><a href="mailto:hadrien.mary AT mcgill DOT ca" target="_blank">Mail: you can use this mail</a></li>
          <li><a href="mailto:hadrien.mary AT gmail DOT com" target="_blank">GMail: or this one</a></li>
          <li><a href="https://github.com/hadim" target="_blank">GitHub: what I code</a></li>
          <li><a href="http://scholar.google.com/citations?user=gGjpl6kAAAAJ" target="_blank">Google Scholar: academic publications</a></li>
          <li><a href="https://twitter.com/hadim_" target="_blank">Twitter</a></li>
      </ul>
    </div><!-- /.social -->
  </section><!-- /#extras -->

  <footer id="contentinfo" class="body">
    <p id="about">
      Powered by <a href="http://getpelican.com/">Pelican</a> -
      Hadrien Mary -
      2014-2016 - <a href="/credits">Credits</a> -
      Last updated: 21/02/2016
    </p><!-- /#about -->
  </footer><!-- /#contentinfo -->

</div>

<script src="/theme/js/jquery-2.2.0.min.js"></script>
<script src="/theme/js/bootstrap.min.js"></script>
<script type="text/javascript">

  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-39149973-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script></body>
</html>